### PR TITLE
feat(starr-fr): French Guide Update

### PR DESCRIPTION
**Pull request by SeiyaGame**
# Pull Request

## Purpose

**[New]**
- [Starr-FR] Add `S4LVE` to `FR Scene Groups`
- [Starr-FR] Add `TFA` to `FR Web Tier 01`
- [Sonarr-FR] Add `IssouCorp` to `FR Anime Tier 2`
- [Sonarr-FR] Add `kazuizui` to `FR Anime Tier 3`

**[Updated]**
- [Starr-FR] Upgrade `FW` from `FR Web Tier 02` to `FR Web Tier 01`
- [Starr] Update `DV` to also match `Dolby V` for french indexer

## Requirements

- [X] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [X] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by SeiyaGame** [comment]
@NiceTSY take a look, just in case

**Comment by SeiyaGame** [comment]
@NiceTSY I also made the changes for Dolby V on the CF DV
#1516

**Comment by NiceTSY** [comment]
Ah thanks for that @SeiyaGame I was planning to do that this weekend :D
Will review this PR tomorrow when I have more time
In the meantime if you could just move `S4LVE` to just after `S0DAP0P` thanks

**Comment by SeiyaGame** [comment]
> Ah thanks for that @SeiyaGame I was planning to do that this weekend :D Will review this PR tomorrow when I have more time In the meantime if you could just move `S4LVE` to just after `S0DAP0P` thanks

Done :D !

File: docs/json/radarr/cf/dv-fel.json
Comment (NiceTSY) [inline docs/json/radarr/cf/dv-fel.json:1]: This is correct but I think the following should be used instead:
`\\b(dv|dovi|dolby[ .]?v(ision)?)\\b`

File: docs/json/sonarr/cf/french-anime-tier-02.json
Comment (NiceTSY) [inline docs/json/sonarr/cf/french-anime-tier-02.json:1]: As we have `FW` in FR Web Tier 02, is it necessary to add it in Anime too?
We suggest using the Tiers normally.

File: docs/json/radarr/cf/dv-fel.json
Comment (SeiyaGame) [reply docs/json/radarr/cf/dv-fel.json:1]: The commit 42ea528 change that and also add those I've forgotten ...

File: docs/json/sonarr/cf/french-anime-tier-02.json
Comment (NiceTSY) [inline docs/json/sonarr/cf/french-anime-tier-02.json:1]: The `IssouCorp` block need to be moved below the `GundamGuy` block to keep the alphabetical order 🙂

File: docs/json/sonarr/cf/french-anime-tier-03.json
Comment (NiceTSY) [inline docs/json/sonarr/cf/french-anime-tier-03.json:1]: The `kazuizui` block need to be moved below the `HANAMi` block to keep the alphabetical order 🙂

File: docs/json/radarr/cf/french-web-tier-01.json
Comment (NiceTSY) [inline docs/json/radarr/cf/french-web-tier-01.json:1]: Sorry, went to fast with this one, `TFA` block need to be moved at the end to keep the alphabetical order

**Review by NiceTSY — CHANGES_REQUESTED** [review]

File: docs/json/sonarr/cf/french-anime-tier-03.json
Comment (SeiyaGame) [reply docs/json/sonarr/cf/french-anime-tier-03.json:1]: Done. (157174c)

File: docs/json/sonarr/cf/french-anime-tier-02.json
Comment (SeiyaGame) [reply docs/json/sonarr/cf/french-anime-tier-02.json:1]: Done. (157174c)

**Comment by SeiyaGame** [comment]
Sorry @TRaSH- For all my mistakes, next time I'll do better.

**Review by NiceTSY — APPROVED** [review]
Everything seems alright for me :) 
Thanks for the hardwork @SeiyaGame

**Comment by SeiyaGame** [comment]
@TRaSH- Ready to merge when you have time

**Pull request merged**